kubernetes_state/check.py

```diff
@@ -204,17 +210,43 @@ def kube_job_complete(self, message, **kwargs):
         for metric in message.metric:
             tags = []
             for label in metric.label:
-                tags.append('{}:{}'.format(label.name, label.value))
+                parsed_job = self._remove_job_tag(label.value)
```

Comment:
Could you please reuse the logic from https://github.com/DataDog/integrations-core/pull/653/files#diff-f11fa8f6e957735347a7525b935dc618R243 to avoid assuming there's only a `job` tag? They might add more later